cineast-core/build.gradle

```diff
@@ -112,6 +112,9 @@ dependencies {
     api group: "com.google.guava", name: "guava", version: version_guava
     api group: "com.googlecode.javaewah", name: "JavaEWAH", version: version_javaewah
 
+    /** Mockito for unit testing */
+    implementation 'org.mockito:mockito-inline:3.11.0'
```

Comment:
Why not mockito-core? https://stackoverflow.com/questions/65986197/difference-between-mockito-core-vs-mockito-inline